### DISCUSSION TITLE
Benefits to use OM buildings in regards to OSM (in FP)

### DISCUSSION THREAD

**Discussion by ViolaineDo** [OvertureMaps/data#66] [Discussion]
Hi 
Thanks for the effort pushing overture map data available. 
I don't see any difference between OSM data and OM data in french polynesia (looking at footprints)
Would it be possible to share an OSM data improved with AI and all instead of only OSM data thks to Overture Map partnership?
Thks in advance

**Comment by jenningsanderson** [comment]
Great question! Yes, we will continue to add AI-derived building footprints where available to the Overture buildings theme. AI-derived building footprints will only appear where there isn't already an OSM building. 

At the moment, however, I don't see any AI-derived building footprints available for French Polynesia in the two major AI-derived building datasets from Microsoft and Google. Do you know of another (open) AI-derived building footprint dataset that might include this region?

**Reply by ViolaineDo** [reply]
Hi,
Thks for the quick answer !
I dont know of any other AI derived building dataset in french polynesia, I was thinking of another option. Would it be possible to ask for building generation through the OM partnership?  
Best regards,
